third_party/build/bin/nettle_build.sh

```diff
@@ -53,7 +53,7 @@ cd ${WORK_DIR}
 wget https://ftp.gnu.org/gnu/nettle/nettle-$PKGVERSION.tar.gz
 tar xf nettle-$PKGVERSION.tar.gz
 cd nettle-$PKGVERSION/
-./configure --disable-openssl --enable-shared --prefix=/usr --build=arm-linux-gnu
```

Comment:
can you detect the arch on the host or pass it as parameter to this build script.

Reply:
Pravin, this change will be abandoned in this PR, but has been put in a PR that has to be merged first. Please review here https://github.com/magma/magma/pull/7409/files#diff-32630ec73120296af0535a109122d18c69968fb2d3e8f69a77660df9ad39e4d4R56-R64